entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Starts the gui inside the docker container using the conda env
+#
+
+# Activate conda env for this script
+# @see https://github.com/ContinuumIO/docker-images/issues/89#issuecomment-467287039
+. /opt/conda/etc/profile.d/conda.sh
+conda activate ldm
+
+# Validate model files
+echo "Validating model files..."
```

Comment:
you can add esrgan models now too it's enabled in environment.yaml

Reply:
I'll try to wrap that all up into a function and pre-download everything. One thing to note is for some of these additional files they will be cached in the `conda_env` volume, so may not even need this additional code/verification to occur.  (I did notice the `GFPGAN` model downloaded automatically when I had initially placed it in the wrong path during testing)

Reply:
nice, let me know when done and i can merge, good work

Reply:
I made some updates which should make things reasonably maintainable and also added the ESRGAN models you mentioned. Adding a more in depth comment below of changes.